test/com/gfredericks/test/chuck/properties_test.cljc

```diff
@@ -0,0 +1,34 @@
+(ns com.gfredericks.test.chuck.properties-test
+  (:require [clojure.test.check :as tc]
+            [clojure.test.check.generators :as gen]
+            [clojure.test.check.clojure-test :as ct
+             #?(:clj :refer :cljs :refer-macros) [defspec]]
+            [com.gfredericks.test.chuck.properties :as prop'
+             #?@(:cljs [:include-macros true])]
+            [com.gfredericks.test.chuck.generators :as gen'
+             #?@(:cljs [:include-macros true])]
+            #?(:clj  [clojure.test :refer :all]
+               :cljs [cljs.test :refer-macros [deftest is]])))
+
+(deftest it-handles-exceptions-correctly
+  (is
+   (instance? #?(:clj  Throwable
+                 :cljs js/Error)
+              (:result
+               (tc/quick-check 100
+                 (prop'/for-all [x gen/int]
+                   #?(:clj  (/ 4 0)
+                      :cljs (js/Error. "Oops"))))))))
```

Comment:
there's gotta be a platform-agnostic way to make something crash :/ what does `(subs nil 1 2)` do in cljs?